### PR TITLE
formik/ENG-4399 profile section

### PR DESCRIPTION
**Pull request by DMinnai**

### CODE REVIEW

**Review by gp-ent — APPROVED** [review]
LGTM :+1:

File: src/ui/common/formik-field/RenderSelectInput.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+  defaultOptionId,
+  options,
+  optionReducer,
+  optionValue,
+  optionDisplayName,
+  size,
+  inputSize,
+  disabled,
+  intl,
+  hasLabel,
```
Comment (ichalagashvili) [inline src/ui/common/formik-field/RenderSelectInput.js]: This file already exists no? Its called `SelectInput`

File: src/ui/common/formik-field/SwitchRenderer.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  };
+
+  return (
+    <div data-testid={dataTestId}>
+      <Switch
+        {...field}
+        value={switchValue}
+        onChange={handleChange}
+      />
+    </div>
```
Comment (ichalagashvili) [inline src/ui/common/formik-field/SwitchRenderer.js]: Does not this already exist in formik-field dir? With different name I think

**Review by ichalagashvili — CHANGES_REQUESTED** [review]
just 2 questions

**Review by ichalagashvili — APPROVED** [review]

**Pull request merged**